cmd/helm/install.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package main
 
 import (
+	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"html/template"
```

Comment:
Oh, yeah... you have to use `sprig.TxtFuncMap()`. Forgot about that.

Otherwise, yes, this is _exactly_ what I had in mind.